src/Symfony/Component/Console/Application.php

```diff
@@ -103,7 +101,8 @@ public function setDispatcher(EventDispatcherInterface $dispatcher)
      *
      * @return int 0 if everything went fine, or an error code
      *
-     * @throws \Exception When doRun returns Exception
+     * @throws \Exception When doRun threw an exception and catchExceptions is false
+     * @throws \Throwable When doRun threw a throwable
```

Comment:
Can be, but the aim was to point the fact throwables are not catch by the application (yet), whereas as this can be configured for exceptions by setting `catchExceptions` to true.

#20808 replicate this behavior for throwables by adding a new `catchThrowables` property.